src/component/Board.jsx

```diff
@@ -125,12 +147,18 @@ function Board() {
               src="https://giphy.com/embed/l0HlSDiA6WUytl9oA"
               className="winnerGif"
             ></iframe>
-            <button className="btn" onClick={restartGame}>
-              Play Again
-            </button>
+            <Button
+              className="btn"
```

Comment:
Component `Button` is not using this prop

Reply:
Implemented.